tests/php/Core/KernelTest.php

```diff
@@ -113,4 +114,16 @@ public function testReplicaDatabaseVarsLoaded()
             'password' => 'hi_people',
         ], $configs['replica_01']);
     }
+
+    public function testImplementorsAreCalled()
```

Comment:
This is the exact test (with some code quality improvements) that used to live in `FlushMiddlewareTest`